lib/vanagon/platform/windows.rb

```diff
@@ -37,12 +37,18 @@ def package_name(project)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_packaging_artifacts(workdir, name, binding)
-        # nuget templates that do require a name change
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/nuget/project.nuspec.erb"), File.join(workdir, "#{name}.nuspec"), false, { :binding => binding })
+        if @package_type == "nuget"
+          # nuget templates that do require a name change
+          erb_file(File.join(VANAGON_ROOT, "resources/windows/nuget/project.nuspec.erb"), File.join(workdir, "#{name}.nuspec"), false, { :binding => binding })
 
-        # nuget static resources to be copied into place
-        ["chocolateyInstall.ps1", "chocolateyUninstall.ps1"].each do |win_file|
-          FileUtils.copy(File.join(VANAGON_ROOT, "resources/windows/nuget/#{win_file}"), File.join(workdir, win_file))
+          # nuget static resources to be copied into place
+          ["chocolateyInstall.ps1", "chocolateyUninstall.ps1"].each do |win_file|
+            FileUtils.copy(File.join(VANAGON_ROOT, "resources/windows/nuget/#{win_file}"), File.join(workdir, win_file))
+          end
+        end
+
+        if @package_type == "msi"
+          erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "service.#{name}.wxs"), false, { :binding => binding })
```

Comment:
@melissa Can you change ```"service.#{name}.wxs"``` to ```"wix/service.#{name}.wxs"```
This means that all the Wix files are placed in a "wix" sub-directory allowing a single copy command to move them to the ```$(tempdir)/wix``` directory.